website/routes.py

```diff
@@ -858,6 +855,30 @@ def make_url_map(app):
             project_views.node.node_registrations,
             OsfWebRenderer('project/registrations.mako', trust=False)
         ),
+        Rule(
+            [
+                '/project/<pid>/registrations/',
+                '/project/<pid>/node/<nid>/registrations/',
+            ],
+            'post',
+            project_views.drafts.new_draft_registration,
+            OsfWebRenderer('project/draft_registration.mako', trust=False)),
+        Rule(
+            [
+                '/project/<pid>/drafts/<draft_id>/',
+                '/project/<pid>/node/<nid>/drafts/<draft_id>/',
+            ],
+            'get',
+            project_views.drafts.edit_draft_registration_page,
+            OsfWebRenderer('project/edit_draft_registration.mako', trust=False)),
+        Rule(
+            [
+                '/project/<pid>/drafts/<draft_id>/register/',
+                '/project/<pid>/node/<nid>/drafts/<draft_id>/register/',
+            ],
+            'get',
+            project_views.drafts.draft_before_register_page,
+            OsfWebRenderer('project/register_draft.mako', trust=False)),
```

Comment:
I was waiting on a response from @brianjgeiger before commenting; Brian,
can we touch base on this?

On Wed, Oct 28, 2015 at 11:36 AM, Andy Boughton <notifications@github.com>
wrote:

> In website/routes.py
> <https://github.com/CenterForOpenScience/osf.io/pull/4428#discussion_r43272131>
> :
>
> > +        Rule(
> > +            [
> > +                '/project/<pid>/drafts/<draft_id>/',
> > +                '/project/<pid>/node/<nid>/drafts/<draft_id>/',
> > +            ],
> > +            'get',
> > +            project_views.drafts.edit_draft_registration_page,
> > +            OsfWebRenderer('project/edit_draft_registration.mako', trust=False)),
> > +        Rule(
> > +            [
> > +                '/project/<pid>/drafts/<draft_id>/register/',
> > +                '/project/<pid>/node/<nid>/drafts/<draft_id>/register/',
> > +            ],
> > +            'get',
> > +            project_views.drafts.draft_before_register_page,
> > +            OsfWebRenderer('project/register_draft.mako', trust=False)),
>
> I know that URLs were sent out for approval. Since I didn't see the
> response, can you please post a confirmation here, as a reply to this
> comment?
>
> (trying to check the review boxes, as it were)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CenterForOpenScience/osf.io/pull/4428/files#r43272131>
> .
>



-- 
Jeffrey Spies
Co-founder/CTO
Center for Open Science
http://cos.io | http://osf.io